docs/root/configuration/overview/examples.rst

```diff
@@ -328,9 +328,10 @@ See the following example:
 
 For example, this will instruct `PyYAML <https://github.com/yaml/pyyaml>` to treat an ignored
 node as a simple scalar when loading:
-.. code-block:: python
+
+.. code-block:: python3
```

Comment:
not sure this is any different from `python` lexer - i think if not just use `python` as that is what is used everywhere else